### PR TITLE
Language optimizations

### PR DESCRIPTION
**Pull request by OzancanKaratas**
- Some upper and lower cases was fixed.

- Use original strings for meaning same translations.

### CODE REVIEW

**Comment by cyBerta** [comment]
Hey @OzancanKaratas, could you change your PR so that it will to merge into deltachat:master instead of deltachat:send-asm? Or is there a specific reason you choose the send-asm feature branch as merge target?

**Comment by cyBerta** [comment]
besides that, it looks good and can be merged

File: tools/untranslated.xml [hunk 1]
```diff
@@ -4,25 +4,24 @@
...
     <string name="reset">Reset</string>
     <string name="edit_contact">Edit contact</string>
-    <string name="custom_port">Custom Port</string>
+    <string name="custom_port">Custom port</string>
     <string name="automatic">Automatic</string>
-    <string name="complete">Complete</string>
+    <string name="completed">Completed</string>
     <string name="info">Info</string>
-    <string name="qr_code_title">QR scan/show</string>
+    <string name="qr_code_title">Scan/Show QR code</string>
```
Comment (cyBerta) [inline tools/untranslated.xml]: `Scan/Show QR code` This string might be too long for the tab bar where it is used

**Comment by r10s** [comment]
thanks for the pr!

one note to `Localizable.strings`: this file is generated, so changes there will be overwritten anyway. (the files is generated from untranslated.xml and from the translations pulled from transifex)

File: deltachat-ios/Controller/ChatViewController.swift [hunk 2]
```diff
@@ -251,7 +251,7 @@ class ChatViewController: MessagesViewController {
 
         if disableWriting {
             menuItems = [
-                UIMenuItem(title: String.localized("start_chat"), action: #selector(MessageCollectionViewCell.messageStartChat(_:))),
+                UIMenuItem(title: String.localized("welcome_start_messaging"), action: #selector(MessageCollectionViewCell.messageStartChat(_:))),
```
Comment (r10s) [inline deltachat-ios/Controller/ChatViewController.swift]: no, welcome_start_messaging is the wrong string here. this string is used on the inital welcome string.

File: tools/untranslated.xml [hunk 3]
```diff
@@ -4,25 +4,24 @@
     <!-- iOS specific untranslated strings -->
     <string name="hide">Hide</string>
     <string name="login_error_title">Unable to login!</string>
-    <string name="start_chat">Start Chat</string>
-    <string name="dismiss">Dismiss</string>
+        <string name="dismiss">Dismiss</string>
```
Comment (r10s) [inline tools/untranslated.xml]: format

**Comment by r10s** [comment]
i think i will close this pr and move the correct string changes to a new pr with a new commit, will credit @OzancanKaratas from there then :)

**Pull request closed without merging**